src/core/service.c

```diff
@@ -3423,18 +3423,7 @@ static void service_notify_message(Unit *u, pid_t pid, char **tags, FDSet *fds)
                 }
         }
 
-        /* Interpret RELOADING= */
-        if (strv_find(tags, "RELOADING=1")) {
-
-                s->notify_state = NOTIFY_RELOADING;
-
-                if (s->state == SERVICE_RUNNING)
-                        service_enter_reload_by_notify(s);
-
-                notify_dbus = true;
-        }
-
-        /* Interpret READY= */
+        /* Interpret READY=/STOPPING=/RELOADING= */
```

Comment:
Wouldn't it be better to take the *last* tag? That seems better in case userspace tries to queue a bunch of tags and they get attached to the same message somehow. The last one seems most relevant. Something like
```C
STRV_FOREACH_REVERSE(tag, tags) {
   if (streq(*tag, "REALODING=1")) {
        ...
        break;
   } else if (streq(*tag, "STOPPING=1")) {
       ...
      break;
};
```